packages/cactus-plugin-ledger-connector-quorum/src/main/json/openapi.json

```diff
@@ -89,9 +89,7 @@
                 "type": "object",
                 "required": [
                     "type",
-                    "ethAccount",
-                    "keychainId",
-                    "keychainEntryKey"
+                    "ethAccount"
```

Comment:
@TonyRowntree I don't get this removal here, could you elaborate?

Reply:
I also don't get this removal, must have been by accident. I will revert the change.